### PR TITLE
gst-vaapi added av1 10bit decode support

### PR DESCRIPTION
**Pull request by FocusLuo**
added av1 10bit decode support

Signed-off-by: Luo Focus <focus.luo@intel.com>

### CODE REVIEW

**Comment by FocusLuo** [comment]
With gst-vaapi av1 decode patch merged and vaapi-fits av1-8bit patch merged, now we can add the 10bit

**Review by uartie — APPROVED** [review]

File: test/gst-vaapi/decode/10bit/av1.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
```
Comment (uartie) [inline test/gst-vaapi/decode/10bit/av1.py:2]: copyright year should be 2021

**Review by uartie — CHANGES_REQUESTED** [review]

File: test/gst-vaapi/decode/10bit/av1.py (hunk 1 above)
Comment (FocusLuo) [reply test/gst-vaapi/decode/10bit/av1.py]: updated, thanks

**Review by uartie — APPROVED** [review]
Thanks. LGTM

**Comment by FocusLuo** [comment]
Thanks for detail review, yes, I also just came back after launch and find this, you are so fast ~~:)

**Pull request merged**